python/lsst/ts/standardscripts/auxtel/mock/attcs_mock.py

```diff
@@ -0,0 +1,296 @@
+        self.athexapod = salobj.Controller("ATHexapod")
+        self.atdometrajectory = salobj.Controller("ATDomeTrajectory")
+
+        self.setting_versions = {}
+
+        self.settings_to_apply = {}
+
+        for comp in self.components:
+            getattr(self, comp).cmd_start.callback = self.get_start_callback(comp)
+            getattr(self, comp).cmd_enable.callback = self.get_enable_callback(comp)
+            getattr(self, comp).cmd_disable.callback = self.get_disable_callback(comp)
+            getattr(self, comp).cmd_standby.callback = self.get_standby_callback(comp)
+
+        self.atdome.cmd_moveShutterMainDoor.callback = self.move_shutter_callback
+        self.atdome.cmd_closeShutter.callback = self.close_shutter_callback
+        self.atpneumatics.cmd_openM1Cover.callback = self.generic_callback
+        self.atpneumatics.cmd_closeM1Cover.callback = self.generic_callback
+        self.ataos.cmd_enableCorrection.callback = self.generic_callback
+        self.ataos.cmd_disableCorrection.callback = self.generic_callback
+
+        self.dome_shutter_pos = 0.
+
+        self.slew_time = 10.
+
+        self.tel_alt = 80.
+        self.tel_az = 0.
+        self.dom_az = 0.
+
+        self.track = False
+
+        self.start_task_finished = False
+        self.start_task = asyncio.create_task(self.start_task_publish())
+
+        self.task_list = []
+
+        self.tel_pos_task = None
+        self.dome_pos_task = None
+        self.run_telemetry_loop = True
+
+        self.atptg.cmd_raDecTarget.callback = self.fake_slew_callback
+        self.atptg.cmd_azElTarget.callback = self.fake_slew_callback
+        self.atptg.cmd_planetTarget.callback = self.fake_slew_callback
+        self.atptg.cmd_stopTracking.callback = self.fake_stop_tracking
+
+        self.atdome.cmd_moveAzimuth.callback = self.fake_move_dome
+
+    async def start_task_publish(self):
+
+        if self.start_task_finished:
+            return
+
+        await asyncio.gather(self.atmcs.start_task,
+                             self.atptg.start_task,
+                             self.atdome.start_task,
+                             self.ataos.start_task,
+                             self.atpneumatics.start_task,
+                             self.athexapod.start_task,
+                             self.atdometrajectory.start_task)
+
+        for comp in self.components:
+            getattr(self, comp).evt_summaryState.set_put(summaryState=salobj.State.STANDBY)
+            self.setting_versions[comp] = f"test_{comp}"
+            getattr(self, comp).evt_settingVersions.set_put(
+                recommendedSettingsVersion=f"{self.setting_versions[comp]},"
+            )
+
+        self.atdome.evt_scbLink.set_put(active=True, force_output=True)
+        self.run_telemetry_loop = True
+        self.tel_pos_task = asyncio.create_task(self.fake_tel_pos_telemetry())
+        self.dome_pos_task = asyncio.create_task(self.dome_tel_pos_telemetry())
```

Comment:
Picky but tthe names are too similar and use confusingly different conventions. Consider `atmcs_telemetry` and `atdome_telemetry` for the methods, and similar names for the tasks.